src/server.ts

```diff
@@ -189,7 +187,7 @@ export async function startServer (port: number, hardware: Hardware = 'v3', com:
       console.log(`Motion ${progress[0] + 1}/${progress[1]}`);
       await new Promise((resolve) => setTimeout(resolve, motion.duration() * 1000));
     },
-    async postCancel(): Promise<void> {
+    async postCancel(_initialPenHeight: number): Promise<void> {
```

Comment:
The 2 implementations should be kept in sync.  (See https://github.com/alexrudd2/saxi/pull/164#issuecomment-2661534659).

Reply:
@alexrudd2 so this is the `simPlotter` object that as far as I understand doesn't implement any action and just logs them. Interestingly the webserial implementation already checks for the pen up original position (or falls back to 50): https://github.com/alexrudd2/saxi/blob/03df63e0f13bd44132d2ce8e0626f5a18c2a1ce4/src/ui.tsx#L185

Reply:
Good point on the `simPlotter.`

Regarding the WebSerial implementation, that's probably the more robust implementation.  The `Math.max` is screwy, but is actually intended to make things work with reversed output pins (where commanding the pen up actually moves it down),